### PR TITLE
add docs about `////` and `///` + small fixes

### PR DESCRIPTION
**Pull request by inoas**

### CODE REVIEW

File: book-src/tour/modules.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Each module has its own namespace and can export types and values to be used
 by other modules in the program.
 
 ```gleam
-// inside src/nasa/rocket_ship.gleam
+//// Inside module src/nasa/rocket_ship.gleam
```
Comment (lpil) [inline book-src/tour/modules.md:8]: This isn't documentation, let's use a normal comment.
```suggestion
// Inside module src/nasa/rocket_ship.gleam
```

File: book-src/tour/modules.md [hunk 2]
```diff
@@ -39,7 +42,7 @@ To use functions or types from another module we need to import them using the
 `import` keyword.
 
 ```gleam
-// inside src/nasa/moon_base.gleam
+//// Inside src/nasa/moon_base.gleam
```
Comment (lpil) [inline book-src/tour/modules.md:45]: This isn't documentation, let's use a normal comment.
```suggestion
// Inside src/nasa/moon_base.gleam
```

**Review by lpil — COMMENTED** [review]
Fab, thank you. Few small comments

File: book-src/tour/functions.md [hunk 3]
```diff
@@ -213,3 +212,26 @@ as the first argument to the call, e.g. `a |> b(1, 2)` would become `b(a, 1, 2)`
...
+
+```gleam
+/// Runs `add()`
+pub fn run() {
+  /// Sums two `Int`s.
+  let add = fn(x, y) { x + y }
+
+  add(1, 2)
+}
+```
```
[the remark this answers is not in the document]
Comment (inoas) [reply book-src/tour/functions.md:237]: Is neither of this valid (function docs, anon functions docs)?
Is `///` thus generally non-valid?

File: book-src/tour/functions.md [hunk 4]
```diff
@@ -213,3 +212,26 @@ as the first argument to the call, e.g. `a |> b(1, 2)` would become `b(a, 1, 2)`
 
 If not it falls back to calling the result of the right hand side as a function
 , e.g. `b(1, 2)(a)`.
+
+## Documentation
+
+Next to explicit type annotation you may add documentation in front of function definitions by prefixing each line of documentation with `///`.
```
[the remark this answers is not in the document]
Comment (inoas) [reply book-src/tour/functions.md:218]: In front of module functions but not anonymous functions?

**Review by lpil — APPROVED** [review]
Thank you

**Pull request merged**